hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoCreateHandler.java

```diff
@@ -418,12 +419,12 @@ public Fees calculateFees(@NonNull final FeeContext feeContext) {
         // Variable bytes plus two additional longs for balance and auto-renew period; plus a boolean for receiver sig
         // required.
         final var op = feeContext.body().cryptoCreateAccountOrThrow();
-        final var keySize = op.hasKey() ? getAccountKeyStorageSize(fromPbj(op.keyOrThrow())) : 0L;
+        final var keySize = op.hasKey() ? getAccountKeyStorageSize(fromPbj(op.keyOrElse(Key.DEFAULT))) : 0L;
```

Comment:
Not a problem, but I'm curious why the change here?

Reply:
Yes, unnecessary. Will change back. I was just removing `throwing`  in `calculateFees`